lib/protoparser/prometheus/parser.go

```diff
@@ -57,19 +57,46 @@ func (rs *Rows) UnmarshalWithErrLogger(s string, errLogger func(s string)) {
 	rs.Rows, rs.tagsPool = unmarshalRows(rs.Rows[:0], s, rs.tagsPool[:0], noEscapes, errLogger)
 }
 
+const tagsPrefix = '{'
+const exemplarPreifx = '{'
+
+/*
+Exemplar Support
```

Comment:
Please, use same comments format as for other struct definitions.
For example: https://github.com/VictoriaMetrics/VictoriaMetrics/blob/ed523b5bbc48543d8e748a4c3204f50acb333044/lib/promscrape/scrapework.go#L46

Comment before struct covers what is this struct intended to represent. Fields have their own comments to clarify the usage/value details